library.properties

```diff
@@ -0,0 +1,10 @@
+name=RESTuino
+version=0.0.1
+author=Yuki Takeyama
+maintainer=takeyamayuki <***@example.com>
+sentence=A firmware for ESP32 to handle arduino GPIO via REST API
+paragraph=A firmware for ESP32 to handle arduino GPIO via REST API
+url=https://github.com/Ar-Ray-code/RESTuino.git
```

Comment:
If the author is me, the URL should also be @takeyamayuki's, but since you have helped me a lot to make the library, so...
Do you want to create a RESTuino organization? Or do you have a better idea?

Reply:
After all, the url should be completely https://github.com/takeyamayuki/RESTuino.git.

If you did pull request to my repository, it should be @takeyamayuki 's URL. If you want @Ar-Ray-code 's URL, please submit the forked repository to arduino as a library.